__tests__/app.test.js

```diff
@@ -54,6 +54,28 @@ describe('Backend testing', () => {
     describe('GET api/articles/:article_id', () => {
     test('status 200: returns article by id', () => {
         const id =1;
+    return request(app)
+      .get(`/api/articles/${id}`)
+      .expect(200)
+      .then(({ body : {article} }) => {
+        expect(article).toEqual(
+            expect.objectContaining({
+                article_id: expect.any(Number),
+                title: expect.any(String),
+                topic: expect.any(String),
+                author:  expect.any(String),
+                body: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number)
+                    })
```

Comment:
The expected article can be more specific as we actually know which one will come back and we know the article_id is 1.